base/applications/mspaint/winproc.cpp

```diff
@@ -452,8 +491,14 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             break;
         }
         case IDM_HELPHELPTOPICS:
-            HtmlHelp(m_hWnd, _T("help\\Paint.chm"), 0, 0);
+        {
+            // NOTE: Don't trust the environment variables
+            WCHAR szPath[MAX_PATH];
+            GetWindowsDirectoryW(szPath, _countof(szPath));
+            wcscat(szPath, L"\\Help\\mspaint.chm");
```

Comment:
The HtmlHelp function that is called by DoHtmlHelp should do this job already by itself.